engine/api/group.go

```diff
@@ -76,6 +78,40 @@ func (api *API) getGroupHandler() service.Handler {
 	}
 }
 
+func (api *API) getGroupExportHandler() service.Handler {
+	return func(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
+		vars := mux.Vars(r)
+
+		groupName := vars["permGroupName"]
+
+		format := FormString(r, "format")
```

Comment:
just an idea: we could respect the accept header for this kind of handler

Reply:
Yes could be nice to plan a change for this.